network/mesh.go

```diff
@@ -229,10 +240,29 @@ func (c hybridRelayMeshCreator) create(opts ...meshOption) (mesher, error) {
 	out := make(chan meshRequest, 5)
 	var wg sync.WaitGroup
 
+	meshFn := func(targetConnCount int) int {
+		wsConnections := cfg.wsnet.meshThreadInner(targetConnCount)
+
+		var p2pConnections int
+		if wsConnections <= targetConnCount {
+			// note "less or equal". Even if p2pTarget is zero it makes sense to call
```

Comment:
Why does it still make sense to make this call?

Reply:
because that's where we have DHT fetching logic.